miniapps/dpg/convection-diffusion.cpp

```diff
@@ -131,7 +131,7 @@ int main(int argc, char *argv[])
    args.Parse();
    if (!args.Good())
    {
-      args.PrintUsage(cout);
+      args.PrintUsage(mfem::out);
```

Comment:
Lets follow what other examples and miniapps do and use `std::cout` -- here and in the parallel version.

Reply:
Another option: this logic (and in the parallel version) can be replaced with the simpler `args.ParseCheck();`

Reply:
@e-aranda, do you mind changing the `mfem::out` with `std::cout`?

Reply:
This issue is now addressed. Thanks @e-aranda!